### PR TITLE
chore: support new directory structure under `orchestrate` dir

### PR DESCRIPTION
**Pull request by pandemicsyn**
Per https://github.com/meltano/hub/pull/822#issuecomment-1276820545 we're reorganizing the on disk layout of airflow config.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,15 +39,15 @@ Meltano Airflow utility extension
       value: false
     - name: database.sql_alchemy_conn
       label: SQL Alchemy Connection
-      value: sqlite:///$MELTANO_PROJECT_ROOT/airflow/airflow.db
+      value: sqlite:///$MELTANO_PROJECT_ROOT/orchestrate/airflow/config/airflow.db
```
Comment (aaronsteers) [inline README.md:42]: Did we revert this decision re the config subfolder?

Comment (pandemicsyn) [reply README.md:42]: No ? This is the subfolder structured you referenced in https://github.com/meltano/hub/pull/822#issuecomment-1276804146:

```
├── meltano.yml
├── orchestrate
│   └── airflow
│       ├── .gitignore
│       ├── dags
│           └── ...
│       ├── logs
│           └── ...
│       ├── plugins
│           └── ...
│       ├── config (or home)
│           └── README.md
│           └── airflow.db
│           └── airflow.cfg
│           └── webserver_config.py
```

The only difference being that webserver_config.py will end up in AIRFLOW_HOME (i.e. `PROJECT_ROOT/orchestrate/airflow`)

Comment (aaronsteers) [reply README.md:42]: Thanks, @pandemicsyn. Just checking. I still like the idea of a `config` subfolder but I guess I thought the constraint on `webserver_config.py` being tied to the `AIRFLOW_HOME` dir also extended to the other config files.
  
Pardon my ignorance on this front. I do like the `config` subfolder being used here; no objection on my part. 👍

File: files_airflow_ext/dot-gitignore [hunk 2]
```diff
@@ -5,7 +5,9 @@
 
 *
 
-# Allow this gitignore, airflow.cfg, webserver_config.py to be checked in to the repo.
+# Allow some explicit files and paths to be checked in
```
Comment (aaronsteers) [inline files_airflow_ext/dot-gitignore:8]: Originally we didn't know much about the files needing to be stored here. If you want to switch the paradigm and specify just the files-to-be-ignored, that's also totally fine, I think.

**Review by aaronsteers — COMMENTED** [review]
Looks good to me! 👍 

If no other issues, I think this is good to go. 🚀

**Review by aaronsteers — APPROVED** [review]

**Pull request merged**